src/thunderbots/software/CMakeLists.txt

```diff
@@ -135,7 +141,24 @@ add_dependencies(backend_output ${catkin_EXPORTED_TARGETS})
 target_link_libraries(backend_output ${catkin_LIBRARIES}
         ${PROTOBUF_LIBRARIES}
         )
+    file(GLOB_RECURSE PARAM_SERVER_SRC LIST_DIRECTORIES false CONFIGURE_DEPENDS
```

Comment:
nit; I think this line shouldn't be indented

Reply:
fixed